client/extensions/woocommerce/state/sites/settings/email/selectors.js

```diff
@@ -38,3 +38,40 @@ export const areEmailSettingsLoaded = ( state, siteId = getSelectedSiteId( state
 export const areEmailSettingsLoading = ( state, siteId = getSelectedSiteId( state ) ) => {
 	return LOADING === getEmailSettings( state, siteId );
 };
+
+const getField = ( field, defaultValue ) => {
```

Comment:
Looks like all current uses of `getField` keep the `defaultValue` to `false`, maybe just set that here and you can remove all the `false` arguments below. Cool code here btw 🙌